src/test/java/edu/cornell/kfs/module/ar/document/validation/impl/CuTotalAmountBilledToDateExceedsAwardTotalSuspensionCategoryTest.java

```diff
@@ -0,0 +1,84 @@
+package edu.cornell.kfs.module.ar.document.validation.impl;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kuali.kfs.krad.document.DocumentBase;
+import org.kuali.kfs.module.ar.businessobject.InvoiceGeneralDetail;
+import org.kuali.kfs.module.ar.document.ContractsGrantsInvoiceDocument;
+import org.kuali.kfs.module.cg.businessobject.Award;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import edu.cornell.kfs.module.cg.businessobject.AwardExtendedAttribute;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ContractsGrantsInvoiceDocument.class})
+public class CuTotalAmountBilledToDateExceedsAwardTotalSuspensionCategoryTest {
+    
+    private CuTotalAmountBilledToDateExceedsAwardTotalSuspensionCategory suspensionCategory;
+    private ContractsGrantsInvoiceDocument contractsGrantsInvoiceDocument;
+    
+    private static final KualiDecimal STANDARD_AWARD_TOTAL = new KualiDecimal(100);
```

Comment:
I don't see this constant being used anywhere in this class. Can it be removed?

Reply:
good catch